goagen/gen_client/cli_generator.go

```diff
@@ -693,4 +732,99 @@ func hasFlag(name string) bool {
 	}
 	return false
 }
-`
+
+func jsonVal(val string) *interface{} {
+	var t interface{}
+	err := json.Unmarshal([]byte(val), t)
+	if err != nil {
+		panic(err)
```

Comment:
Yeah bubble up all the way to `Run` so that the failure is consistent with e.g. a malformed payload JSON.